GNUmakefile

```diff
@@ -0,0 +1,32 @@
+MKDIR_P := mkdir -p
+RM_F := rm -rf
+
+export GO111MODULE=on
+
+Targets := \
+	echoserver \
+	ingress-controller-conformance
+
+build: $(Targets) ## Build the conformance tool
+
+.PHONY: echoserver
```

Comment:
if echoserver is a file on the file system, why make this phony? Also, shouldn't we put the dependent sources as deps

Reply:
Teaching make about Go's notion of dependencies isn't really practical, so it's better to mark the targets as phony so that Go will always get a chance to rebuild if it's necessary.